bw-dev

```diff
@@ -123,6 +123,9 @@ case "$CMD" in
     populate_streams)
         runweb python manage.py populate_streams
         ;;
+    generateimages)
```

Comment:
To avoid confusion with `generate_preview_images`, would `generate_thumbnails` be clearer here?

Reply:
You're right, it'll be less confusing that way.

The command `generateimages` comes from `django-imagekit` (https://github.com/matthewwithanm/django-imagekit/#management-commands), but it'll be more consitent internally (and less confusing) to use `generate_thumbnails`, I'm changing it rignt now.